migrations/20160606164500-add-expenses-table.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+
+const status = require('../server/constants/expense_status');
+
+module.exports = {
```

Comment:
Well, the model `Expense` is already checked in and pushed. I thought it would have given an error when it doesn't find the corresponding table in the database?